roles/installer/molecule/default/converge.yml

```diff
@@ -4,7 +4,13 @@
   gather_facts: false
   vars:
     installer_scenario: foreman
-    installer_options:
-      - '--foreman-initial-admin-password changeme'
   roles:
-    - installer
+    - role: installer
+      vars:
+        installer_options:
+          - '--foreman-initial-admin-password changeme'
+    - role: installer
+      vars:
+        installer_options:
+          - '--foreman-initial-admin-password changeme'
```

Comment:
Technically, we don't need to pass the password anymore.

Reply:
more technically, we don't need to set the pass in the first call either, nobody cares for the password in the container that is thrown away after the installer run anyways.

Reply:
So why are they in here? :)

Reply:
Because that was not the part I was changing, and thought we could come to a consensus before nuking :)

Reply:
I think I was anticipating re-use of this down the line where password might be used. You can nuke it.